book/leerdoelen/onzekerheidToleranties.md

```diff
@@ -67,3 +67,41 @@ Bij het natuurkundig practicum was de uitdaging om bij een gegeven opstelling te
 
 Bij het ontwerpen van een meet-apparaat of een meet-opstelling worden dezelfde vergelijkingen gebruikt om een ander probleem op te lossen, namelijk: hoe nauwkeurig moeten onderdelen van mijn oplossing zijn om een gewenste nauwkeurigheid te behalen. De maximaal toegelaten nauwkeurigheid ($U\left( Z\right)$ in bovenstaande vergelijkingen) is dan gegeven. De op te lossen vraag is welke combinatie van ontwerpkeuzes ervoor zorgt dat de opstelling aan dit criteria gaat voldoen. Realiseer je daarbij dat een opstelling relatief nauwkeuriger maken vaak zowel kan door $U\left( A\right)$ te verlagen als door $A$ te vergroten.
 
+### Voorbeeld: Maximaal vermogen
+Stel je hebt een simpele schakeling van een LEDje, een weerstand en een variabelle spanningsbron. Het maximale vermogen dat door de schakeling mag is bekend, daarboven gaat de schakeling kapot. De spanningsval over de LED is 2 Volt, de spanningsbron heeft een bereik van 0 tot 220 Volt en een onzekerheid van 1 Volt. Je hebt weerstanden met een nauwkeurigheid van 1% of 5% tot je beschikking. Je wilt dat de onzekerheid in het vermogen lager dan 10% is zodat je niet per ongelukt de schakeling kapot maakt. Deel het bereik van de spanningsdeler op en geef voor elk gebied aan welke weerstand je moet kiezen.
```

Comment:
ik weet niet of ze weten wat een spanningsval is

Reply:
goed punt, haal ik eruit.